layouts/partials/kubernetes-overview.html

```diff
@@ -0,0 +1,31 @@
+{{- $shouldShowBirthdayBanner := false -}}
+{{- $now := time.Now -}}
+{{- with site.GetPage "page" "community/special/kubernetes-10th-birthday" -}}
+  {{- if and (lt ( (.Param "display_date_start") | time.AsTime ) $now ) (gt ( (.Param "display_date_end") | time.AsTime ) ( $now.AddDate 0 0 1 ) ) -}}
+    {{- $shouldShowBirthdayBanner = true -}}
+    {{- warnf "%s" "It's Kubernetes' birthday, fingers crossed this renders OK" -}}
```

Comment:
I don't mind having this warning render for the next 10 days

Reply:
This is cute